### DISCUSSION TITLE
A solution for more than one word in a slug

### DISCUSSION THREAD

**Discussion by invictivi** [velsa/notehost#30] [Ideas]
Hello folks,

I encountered a challenge when working on my website.
I wanted to have a slug that matches the slug in Notion, with dashes as separators.
Unfortunately, I got an error from the `src/site-config.ts`, dashes are not expected to appear there.

It would be amazing to have a solution for that :)

I am not a programmer or DevOps person. I like to do my stuff on my own and I just lack the skills to do that.

**Comment by velsa** [comment]
Hi! 

Dashes and slashes are supported in slug names.
Not sure what error you get. 

Here's a test I did on my own site:

config:

```js
    'te-st': '3af863c2801742f28f154e55534cc72c',
    'te-st/ano-ther-test': '6f7547962d7643e3bd450ddf2f4cc6ff',
```

And here it is deployed:

https://velsa.net/te-st
https://velsa.net/te-st/ano-ther-test
